lib/generators/scenic/model/model_generator.rb

```diff
@@ -8,6 +8,7 @@ module Generators
     # @api private
     class ModelGenerator < Rails::Generators::NamedBase
       include Scenic::Generators::Materializable
+
```

Comment:
The code formatter added these. I guess I could have avoided committing the unrelated changes, but it's the results of following the contributing guide...